metrics/stats.go

```diff
@@ -0,0 +1,161 @@
+package metrics
+
+import (
+	"runtime"
+	"sync"
+	"sync/atomic"
+	"time"
+)
+
+type Stats struct {
+	Time int64 `json:"time"`
+	// runtime
+	GoVersion    string `json:"go_version"`
+	GoOs         string `json:"go_os"`
+	GoArch       string `json:"go_arch"`
+	CpuNum       int    `json:"cpu_num"`
```

Comment:
__[golint]__
* [golint] struct field CpuNum should be CPUNum

<!-- Comment by [SideCI](https://sideci.com) -->